src/Controls/src/Core/Platform/AlertManager/AlertManager.Android.cs

```diff
@@ -255,7 +256,7 @@ void OnPromptRequested(IView sender, PromptArguments arguments)
 				alertDialog.SetMessage(arguments.Message);
 
 				var frameLayout = new FrameLayout(Activity);
-				var editText = new EditText(Activity) { Hint = arguments.Placeholder, Text = arguments.InitialValue };
+				var editText = new AppCompatEditText(Activity) { Hint = arguments.Placeholder, Text = arguments.InitialValue };
```

Comment:
@jfversluis I found this one as well. 

Looks like we need it done in XF as well?
https://github.com/xamarin/Xamarin.Forms/blob/5.0.0/Xamarin.Forms.Platform.Android/PopupManager.cs#L204

Reply:
There is another one around line 193 in the `AlertManager`

Reply:
That's no instantiating a `TextView` it's just extracting it from the Dialog. I inspected the `TextView` and the `TextView` pulled from the dialog is a `MaterialTextView` so we should be fine here